synology_api/core_certificate.py

```diff
@@ -88,7 +88,7 @@ def upload_cert(self,
 
         if cert_id:
             print("update exist cert: " + cert_id)
-        data_payload = {'id': cert_id or '', 'desc': desc or '', 'as_default': 'true' if set_as_default else 'false'}
+        data_payload = {'id': cert_id or '', 'desc': desc or '', 'as_default': 'true' if set_as_default else ''}
```

Comment:
Alternatively, this could avoid setting `set_as_default` when false:

```python
        data_payload = {'id': cert_id or '', 'desc': desc or ''}
        if set_as_default:
            data_payload['as_default'] = 'true'
```

Both work, in my testing.